### PR TITLE
Match maker

### PR DESCRIPTION
**Pull request by lvgilmore**
opening pull request for review purpose

### CODE REVIEW

**Review by lvgilmore — COMMENTED** [review]
I think I'm missing something... you initialize Request with default values, but you never change the values and you only save to the file the json representation?

Regarding the ConfigParser, I don't fully get it, but I trust you that it works. I'll use it

**Comment by yuliaTev** [comment]
When new bare metal comes, i read the values from file to list and initial the Request according to it.
The initialization happens when i read from a file, not before it.

File: src/utils/JsonUtils.py [hunk 1]
```diff
@@ -1,55 +1,79 @@
...
+
+    for request in requests:
+        requests_list_obj.append(parse_req(request))
+
+    return requests_list_obj
+
+
+def parse_req(request):
+    req = Request(request)
+    for key in request.keys():
```
Comment (yuliaTev) [inline src/utils/JsonUtils.py:48]: for key in ...:
    req.requirements.**dict**[key] = ...

File: src/matchMaker/MatchMaker.py [hunk 2]
```diff
@@ -1,58 +1,99 @@
...
-        elif key == "url":
-            score = 4
+        return matched_req_by_requirements
+
+    # TODO decide what we prefer: there is another option, to use sections, and declare a default section,
+    #  so when the score is not declared, it will search it in defaule section and init score with 0
+    def calc_score(self, key):
+        score = 0
+
+        try:
```
Comment (yuliaTev) [inline src/matchMaker/MatchMaker.py:85]: I think we should override get, so we won't need to know about SECTION and about what to except

**Pull request merged**